spec/decorators/order_decorator_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+RSpec.describe OrderDecorator do
+  let!(:order) { create(:order, :order_items).decorate }
+
+  describe '#subtotal' do
+    let(:result) { order.subtotal }
+    let(:expected_result) { order.order_items.sum { |item| item.quantity * item.book.price } }
+
+    it 'give subtotal money for order_items' do
+      expect(result).to eq(expected_result)
+    end
+  end
+
+  describe 'functionality with coupon' do
+    let(:max_discount) { Coupon::MAX_DISCOUNT }
+    let(:subtotal) { order.order_items.sum { |item| item.quantity * item.book.price } }
```

Comment:
just pass correct number here, do not duplicate code from real method

Reply:
fixed